MetamodelAlgemeen.md

```diff
@@ -309,6 +309,12 @@ Voorbeelden van gegevens zijn de waardes ‘Jan’ en ‘1-1-1970’ betreffende
 Gegevens zijn de objectief waarneembare neerslag of registratie van feiten op een bepaald medium, zodanig dat deze gegevens uitgewisseld en voor langere tijd bewaard kunnen worden. Dat kan op papier, in digitale vorm, et cetera. Met deze gegevens wordt een model (een selectief deel dus) van de werkelijkheid vastgelegd in de tijd. Ofschoon de werkelijkheid nooit stilstaat, kan deze door het vastleggen van de gegevens toch worden bevroren. 
 Het waargenomen feit dat twee objecten een relatie hebben wordt ook gezien als een gegeven.
 
+Zodra een gegeven vastgelegd wordt of uitgewisseld wordt noemen we deze ook wel data. Het is deze data die waarde/data van een eigenschap van een object is, bv. 'Jan' is de waarde/data van de eigenschap voornaam van een Natuurlijk persoon. Data aangevuld met de betekenis uit het informatiemodel, waarin de definitie van wat een voornaam is staat, noemen we informatie.
```

Comment:
> 

Ik kan me hier niet helemaal in vinden. De waarde van een eigenschap van een object is m.i. onafhankelijk van data. Een waarde is onderdeel van een gegeven. Zonder waarde is er geen gegeven. Een gegeven is een toekenning van een waarde van een kenmerksoort aan een object. Een gegeven kan bestaan zonder dat er bijbehorende data van bestaat. 

Als we een gegeven systematisch willen beschrijven en willen kunnen onderscheiden van andere gegevens hebben we volgens mij drie onderdelen nodig:

een object (het onderwerp van het gegeven)
een kenmerksoort
een waarde

> Data aangevuld met de betekenis uit het informatiemodel, waarin de definitie van wat een voornaam is staat, noemen we informatie.

Dit vind ik een lastige stelling. Hier kunnen we nog een hele filosofische discussie aan wijden. Het lijkt me verder wel goed om informatie te definiëren, maar dan moet het ook een prominentere plek krijgen in het document.

Reply:
NEN3610 definieert een aantal van deze begrippen al. Hoe logisch/zinvol is het om in het MIM nieuwe (varianten op die) definities te hanteren?

Reply:
Ik weet niet of ik dit deel van de aanvulling nodig vindt. Volgens mij staat het er zonder deze toevoeging ook best helder.

Reply:
> Ik weet niet of ik dit deel van de aanvulling nodig vindt. Volgens mij staat het er zonder deze toevoeging ook best helder.

Ik denk ook niet dat het nodig is. Volgens mij is dit toegevoegd om de term data te definiëren, die dan weer later in het stuk over de interne structuur van gegevens gebruikt wordt: "data element". Daar gaat het volgens mij niet over data, maar over de waarde, en diens interne structuur.